db/migrate/20150717135909_remove_unique_index_from_builds.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUniqueIndexFromBuilds < ActiveRecord::Migration
+  def change
+    # Remove unique index on git_sha, users should be able to rebuild docker images against same SHA
```

Comment:
ENV vars are not baked into the docker image.  You build the docker image once and push it to the registry, and then use the same image for all deployments (master, staging, gamma, pod1-6, etc.) Environment variables are injected when the image is run in a container.

When Samson builds an image, it effectively does this:

```shell
git checkout $SHA
echo $SHA > REVISION
docker build -t $DOCKER_TAG .
docker push $DOCKER_TAG
```

So the contents of the image are based solely on the contents of the repo. Unless we changed the logic on how we generate the builds (like removing the REVISION file), building an image twice should always generate the exact same image.

That make sense?